assets/js/carousel-swipe.js

```diff
@@ -0,0 +1,46 @@
+var drinksCarousel = document.getElementById('drinks-carousel');
+var app;
+if (drinksCarousel) {
+  app = Elm.DrinksCarousel.init({
+    node: drinksCarousel
+  })
+}
+
+function swipe(el, callback){
```

Comment:
please add JSDOC comments to JavaScript code so others can understand it. 
ideally state the _source_ of the code.